tests/infra/remote.py

```diff
@@ -501,15 +502,19 @@ def __init__(
                 auto_dr_args["previous_sealed_ledger_secret_location"] = (
                     previous_sealed_ledger_secret_location
                 )
+
+            enclave_platform = infra.platform_detection.get_platform()
+            enclave_platform = (
+                "Virtual"
+                if enclave_platform.lower() == "virtual"
+                else enclave_platform.upper()
```

Comment:
Seems simpler to have get_platform() return "Virtual"/"SNP",  and to lower() if/where needed at all?

Reply:
I've done it this way round because this specific casing is the legacy values (constructing this `enclave_platform` value which is only needed for LTS compat, we don't even parse it in `7.x` nodes), so the more-common place where we might ever have to spell/case this correctly is when trying to set an override value. I'd prefer to type `CCF_PLATFORM_OVERRIDE=virtual` (rule is "they're all lower-case") than remembering the exact casing. That mirrors the current use, where we mindlessly type `cmake ... -DCOMPILE_TARGET=virtual`, and only get bitten by `Virtual|SNP` when trying to build manual configurations or lower-level Python args.

Reply:
Summary: I'd prefer to keep this as currently written.